docs/preview-apis.md

```diff
@@ -85,6 +94,6 @@ framework as preview API for a while.
 
 We've tentatively set .NET 11 as the release when we'll stop marking FSKit as preview API.
 
-The diagnostic id for CryptoTokenKit is APL0001.
```

Comment:
What happened to CryptoTokenKit?

Reply:
Nothing, this was a copy/paste mistake when the section about FSKit was added.